src/configuration/Configuration.h

```diff
@@ -46,20 +48,28 @@ namespace SlimeVR {
             CalibrationConfig getCalibration(size_t sensorID) const;
             void setCalibration(size_t sensorID, const CalibrationConfig& config);
 
+            size_t getWiFiCredentialCount() const;
+            const std::map<const char*, WiFiCredential>* const getWiFiCredentials() const;
```

Comment:
why not `unordered_map`?

Reply:
doesn't compile on esp32

Reply:
I just tested this on both `esp12e` and `esp32`, both compile. `unordered_map` was added in C++ 11 so it should work fine.